exporters/user_events/src/utils.cc

```diff
@@ -0,0 +1,87 @@
+
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#include "opentelemetry/exporters/user_events/logs/utils.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+
+namespace exporter
+{
+namespace user_events
+{
+namespace utils
+{
+
+namespace api_common = opentelemetry::common;
+
+const int kAttributeValueSize = 16;
+
+void PopulateAttribute(nostd::string_view key,
+                       const api_common::AttributeValue &value,
+                       ehd::EventBuilder &event_builder) noexcept
+{
+  static_assert(nostd::variant_size<api_common::AttributeValue>::value == kAttributeValueSize,
+                "AttributeValue has changed, update PopulateAttributeValue");
+
+  const char *key_name = key.data();
+
+  if (nostd::holds_alternative<bool>(value))
```

Comment:
This looks like a std::variant, so maybe there is a `visit` function for it. If there is a `visit` function, it's usually a good idea to use it -- it's usually more efficient than an IF-ELSE chain (it usually compiles into a SWITCH statement), and it helps you make sure you don't accidentally miss any cases.